.github/workflows/ci.yml

```diff
@@ -24,35 +23,42 @@ jobs:
             ghc: 8.8.4
           - os: macOS-latest
             ghc: 8.6.5
-          - os: windows-latest
-            ghc: 8.8.4
           - os: windows-latest
             ghc: 8.10.2
+          - os: windows-latest
+            ghc: 8.6.5
 
     steps:
     - uses: actions/checkout@v2
       if: github.event.action == 'opened' || github.event.action == 'synchronize' || github.event.ref == 'refs/heads/main'
 
-    - uses: actions/setup-haskell@v1.1.3
+    - uses: actions/setup-haskell@v1.1.4
       id: setup-haskell-cabal
       name: Setup Haskell
       with:
         ghc-version: ${{ matrix.ghc }}
         cabal-version: ${{ matrix.cabal }}
 
+    - name: Configure
+      run: |
+        cabal configure --enable-tests --enable-benchmarks --test-show-details=direct
```

Comment:
Apparently, this needs to be called before `freeze`, otherwise `freeze` sometimes doesn't include dependencies for test-suite (~because ****~, because `--enable-tests` is not specified for `freeze`). So a quick fix is to move `configure` before all commands